packages/plugin-types/API.md

```diff
@@ -186,7 +186,6 @@ export type PackageManagerController = {
     installDependencies: (packageNames: string[], type: 'dev' | 'prod') => Promise<void>;
     runWithPackageManager: (args: string[] | undefined, dir: string, options?: Options<'utf8'>) => ExecaChildProcess;
     getCommand: (args: string[]) => string;
-    allowsSignalPropagation: () => boolean;

```

Comment:
we shouldn't be removing public api. it's possible to install older cli with new plugin-types.